docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -1,143 +1,72 @@
 [[synthetic-run-tests]]
-= Run synthetic tests
+= Create a synthetic monitor
 
-++++
-<titleabbrev>Run synthetic tests</titleabbrev>
-++++
+Once you've written a synthetic test, you can create a monitor to run the test on a regular interval.
 
-There are two ways to run synthetic tests:
+// There are two ways to run synthetic tests:
 
-* If you have more than one journey, rely on dependencies, or your tests need to live with your
-application code, <<synthetics-test-suite,use a test suite>>.
-* If you want to create a single journey and manage it individually, <<synthetics-inline-journey,use an inline monitor>>.
+// * If you have more than one journey, rely on dependencies, or your tests need to live with your
+// application code, <<synthetics-test-suite,use a test suite>>.
+// * If you want to create a single journey and manage it individually, <<synthetics-inline-journey,use an inline monitor>>.
```

Comment:
Do we still want to talk about suite/projects and inline monitors at the same level or do we want to emphasize that the suite/project approach is the primary workflow, but inline monitors are possible?